resources/views/components/game/top-achievers/score-row.blade.php

```diff
@@ -15,5 +16,12 @@
             <x-user.avatar display="icon" icon-size="xs" :user="$user" />
             <x-user.avatar hasHref="true" :user="$user" />
         </td>
-        <td class="text-right">{{ localized_number($score) }}</td>
+        <td>
+            <div class="flex items-center gap-1 float-right">
```

Comment:
With a flex layout, "justify-end" is preferable to "float-right".

This doc is a great explainer on flex justify with helpful infographics: https://tailwindcss.com/docs/justify-content